Makefile

```diff
@@ -95,7 +95,13 @@ sdist:
 .PHONY: test
 test:
 	python3 -c "import pytest" > /dev/null 2>&1 || python3 -m pip install pytest
-	python3 -m pytest -qq
+	python3 -m pytest -qq Tests/
```

Comment:
Why this change?

Reply:
A minor attempt to speed up test discovery.